signer/src/testing/storage/model.rs

```diff
@@ -62,6 +63,10 @@ fn idealistic_bitcoin_chain(
 ) -> impl FnMut(&mut Vec<model::BitcoinBlockHash>, usize) -> Option<model::BitcoinBlock> + '_ {
     |block_hashes, _| {
         let mut block: model::BitcoinBlock = fake::Faker.fake_with_rng(rng);
+        let block_hash: [u8; 32] = fake::Faker.fake_with_rng(rng);
+        block.block_hash = block_hash.to_vec();
+        block.created_at = block.created_at.max(OffsetDateTime::now_utc());
```

Comment:
I would like this function to be deterministic. If we need to control the `created_at`, I'd prefer another utility to do so - or simply just modify the created blocks after calling this function.

Reply:
Oh, right whoops. I can make it deterministic. I don't know why I didn't think to use the unix epoch before.